roles/common/tasks/ssl.yml

```diff
@@ -4,7 +4,7 @@
 
 - name: (possibly self-signed) ssl cert
   template: src=openstack.crt
-            dest=/usr/local/share/ca-certificates/{{ endpoints.main }}.crt
+            dest=/usr/local/share/ca-certificates/{{ fqdn }}.crt
```

Comment:
is this needed if you make endpoints a role dep? Why wouldnt' we use endpoints.main?